ui/pages/AnvilWorkspace/components/LoadWorkspaceDataForm.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Header, Segment } from 'semantic-ui-react'
+
+import { FILE_FIELD_NAME, PROJECT_DESC_FIELD, GENOME_VERSION_FIELD, FAMILY_FIELD_ID, INDIVIDUAL_FIELD_ID } from 'shared/utils/constants'
+import { BaseBulkContent, BASE_UPLOAD_FORMATS } from 'pages/Project/components/edit-families-and-individuals/BulkEditForm'
+import { INDIVIDUAL_CORE_EXPORT_DATA, INDIVIDUAL_ID_EXPORT_DATA } from 'pages/Project/constants'
+import ReduxFormWrapper from 'shared/components/form/ReduxFormWrapper'
+import { BooleanCheckbox } from 'shared/components/form/Inputs'
+
+const FIELD_DESCRIPTIONS = {
+  [FAMILY_FIELD_ID]: 'Family ID',
+  [INDIVIDUAL_FIELD_ID]: 'Individual ID (needs to match the VCF ids)',
+}
+const REQUIRED_FIELDS = INDIVIDUAL_ID_EXPORT_DATA.map(config => (
+  { ...config, description: FIELD_DESCRIPTIONS[config.field] }))
+
+const BLANK_EXPORT = {
+  filename: 'individuals_template',
+  rawData: [],
+  headers: [...INDIVIDUAL_ID_EXPORT_DATA, ...INDIVIDUAL_CORE_EXPORT_DATA].map(config => config.header),
+  processRow: val => val,
+}
+
+const UploadPedigreeField = React.memo(() =>
+  [
+    // eslint-disable-next-line jsx-a11y/label-has-for
+    <label key="uploadLabel">Upload Pedigree Data</label>,
+    <Segment key="uploadForm">
+      <BaseBulkContent
+        blankExportConfig={BLANK_EXPORT}
+        requiredFields={REQUIRED_FIELDS}
+        optionalFields={INDIVIDUAL_CORE_EXPORT_DATA}
+        uploadFormats={BASE_UPLOAD_FORMATS}
+        actionDescription="load individual data from an AnVIL workspace to a new seqr project"
+        url="/api/upload_temp_file"
+      />
+    </Segment>,
+  ],
+)
+
+const UPLOAD_PEDIGREE_FIELD = {
+  name: FILE_FIELD_NAME,
+  label: 'Upload Pedigree Data',
+  component: UploadPedigreeField,
+}
+
+const AGREE_CHECKBOX = {
+  name: 'agreeCheckbox',
+  component: BooleanCheckbox,
+  label: 'By submitting this form I agree to grant seqr access to the data in the associated workspace',
+}
+
+const FORM_FIELDS = [PROJECT_DESC_FIELD, UPLOAD_PEDIGREE_FIELD, GENOME_VERSION_FIELD, AGREE_CHECKBOX]
+
+const LoadWorkspaceDataForm = React.memo(({ namespace, name }) =>
+  <div>
+    <Header className="large center aligned">Load data to seqr from AnVIL Workspace &quot;{namespace}/{name}&quot;</Header>
```

Comment:
please read the documentation for components you are adding (https://react.semantic-ui.com/elements/header/) or look at other places in the code to see how they are done. If you still can't figure it out, ask me for help. We NEVER pass `className` to our semantic UI components. We pass the props we want to the component  and let them compute the classes. If you read the documentation it will tell you what props are available. This is an important thing for you to learn to do so I am not going to tell you which props you want ehre. If you can't figure it out from the documentation on your own, set up a meeting with me and I will show you how to do it for next time